command-line/README.md

```diff
@@ -6,11 +6,11 @@ Build jar with
 ./gradlew :command-line:noDepsJar
 ```
 
-Get it at `command-line/build/libs/helium-cli.jar`.
+Get it at `command-line/build/libs/command-line-{version}-nodeps.jar`.
```

Comment:
It's a bit inconvenient now because it now matches Maven naming conventions :)
But once we have the script, it will not matter.